site/releases/2024-sep-09/release-notes.qmd

```diff
@@ -0,0 +1,138 @@
+
+<form method="get" action="/guide/model-validation/manage-validation-guidelines.html" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">Manage validation guidelines</button>
+</form>
+
+:::
+
+::: {.w-50-ns}
+
+![](settings-validation-guidelines.png){fig-alt="A screenshot showing the Risk Areas & Validation Guidelines in Workspace Settings"}
+
+:::
+
+::::
+
+
+<!---
+PR #921: Added "Reset Workflow" on the model inventory page
+URL: https://github.com/validmind/frontend/pull/921
+Labels: enhancement
+--->
+
+### Reset your model workflow
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+
+Need to start over with a model? You can now reset your model's workflow to its initial state in the lifecycle.
+
+<form method="get" action="/guide/model-workflows/working-with-model-workflows.html#reset-your-model-workflow" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">Working with model workflows</button>
+</form>
+
+:::
+
+::: {.w-30-ns}
+
+![](reset-model-workflow.png){fig-alt="A screenshot showing the option to reset your model workflow"}
+
+
+:::
+
+::::
+
+## Enhancements
+
+### Model inventory fields
+
+Previously, we used custom fields to refer to your organization's unique additional fields for models as well as the default fields provided as suggestions by ValidMind. Custom fields has been renamed to model inventory fields, capturing all additional fields, and the terminology standardized across the {{< var validmind.platform >}} and our documentation.
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+You can still require additional fields to be provided upon model registration, and work with a number of different types of fields.
+
+:::
+
+::: {.w-30-ns}
+
+<form method="get" action="/guide/model-inventory/manage-model-inventory-fields.html" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">Manage model inventory fields</button>
+</form>
+
+:::
+
+::::
+
+
+
+<!---
+PR #929: Renamed Statuses to Model Lifecycle Statuses
+URL: https://github.com/validmind/frontend/pull/929
+Labels: enhancement
+--->
+### Model lifecyle statuses
+
+Previously, we resource statuses to refer to the model stages made available for use in your model workflows. Resource statuses has been renamed to model lifecycle statuses, clarifying their function, and the terminology standardized across the {{< var validmind.platform >}} and our documentation.
```

Comment:
Yes, this is because there needs to be a summary of the contents at each page for the round-up listing cards: https://docs-demo.vm.validmind.ai/pr_previews/beck/sc-6089/release-notes-to-sprint-57/releases/2024-releases.html

Reply:
We're generally in favour of less words, not more. If there's a simpler way to say it, we do that.

Reply:
Fewer words is better.